src/main/java/twilightforest/TFEventListener.java

```diff
@@ -680,22 +686,23 @@ public static void onServerDisconnect(PlayerEvent.PlayerLoggedOutEvent event) {
 	public static void worldLoaded(WorldEvent.Load event) {
 		World world = event.getWorld().getWorld();
 
-		if (!world.isRemote() && !world.getGameRules().get(TwilightForestMod.ENFORCED_PROGRESSION_RULE)) {
+		if (!world.isRemote() && world.getGameRules().get(TwilightForestMod.ENFORCED_PROGRESSION_RULE) != null) {
```

Comment:
Just a general question.. is this actually proper..? Is the game rule always null when false? Just seems weird.